app/api/services/suppliers.py

```diff
@@ -22,6 +22,10 @@ class SuppliersService(Service):
     def __init__(self, *args, **kwargs):
         super(SuppliersService, self).__init__(*args, **kwargs)
 
+    def get_all_suppliers(self):
+        results = (db.session.query(Supplier).all())
```

Comment:
This query won't perform well.  We should only fetch what data we need for these reports.

Reply:
Not sure about this the requirement say 'all' I assume all seller.

Reply:
modified  the query retrieve data for  specific fields